modules/administration/pages/custom-channels.adoc

```diff
@@ -0,0 +1,78 @@
+[[custom-channel]]
+= Custom Channels
+
+While {productname} provides all required channels, you might find it useful to create custom channels specific to your environment.
+{productname} Administrators and Channel Administrators have channel management authority, which gives them the ability to create and manage their own custom channels.
+
+Before you create a custom channel, determine which base channel you want to associate it with, and which repositories you want to use for content.
+
+This section gives more detail on how to create, administer, and delete custom channels.
+
+
+
+== Creating Custom Channels
+
+If you have custom software packages that you need to install on your {productname} systems, you can create a custom child channel to manage them.
+You will need to create the channel in the {productname} {webui} and create a repository for the packages, before assigning the channel to your systems.
+
+You can select a vendor channel as the base channel if you want to use packages provided by a vendor.
+Alternatively, select ``none`` to make your custom channel a base channel.
+
+If you include a package with [systemitem]``-release`` at the end of the package name in your channel, it will be automatically installed when the channel is installed.
```

Comment:
```suggestion
If you include a package with [systemitem]``-release`` at the end of the package name in your channel, it will be automatically installed when the channel is assigned.
```